src/shared/utils/browser/set-auth-cookie.ts

```diff
@@ -0,0 +1,16 @@
+import { isHttps } from "@utils/env";
+import * as cookie from "cookie";
+import { authCookieName } from "shared/config";
+
+export default function setAuthCookie(jwt: string) {
+  const expires = new Date();
+  expires.setDate(expires.getDate() + 365);
+
+  document.cookie = cookie.serialize(authCookieName, jwt, {
+    expires,
+    secure: isHttps(),
+    domain: location.hostname,
```

Comment:
Should probably get rid of domain. Even if it's not causing the issue, I don't think we need it.